internal/pkg/mapper/sharedcode/links/load.go

```diff
@@ -0,0 +1,120 @@
+package links
+
+import (
+	"fmt"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils"
+)
+
+func (m *mapper) OnObjectsLoad(event model.OnObjectsLoadEvent) error {
```

Comment:
Prejdu sa vsetky konfiguracie, ktore maju lokalne ulozene `shared_code_path`, a interne sa to nahradi za `shared_code_id`.